typings/blocks.d.ts

```diff
@@ -11,5 +11,4 @@
 
 /// <reference path="core.d.ts" />
 
-import * as Blockly from './core';
-export = Blockly.Blocks;
+export { Blocks } from './core';
```

Comment:
Hm…  I think this is wrong.  If I understand correctly, this file is supposed to be typings for the stuff in `blocks/`—i.e., the `Blockly.libraryBlocks` modules—not the single dictionary object exported by `core/blocks.js`.

Notably, while I think it was believed that `import * as Blocks from 'blockly/blocks'` would get you a copy of the `Blockly.Blocks` dictionary object, that was AFAIK not in fact the case (due to bugs) at the time we transitioned to `goog.module`, if ever.

In any case, the blocks modules got their own more useful set of exports via PRs #5900 and #5908, and were renamed to `Blockly.libraryBlocks` in #5953.  Some appropriate typings should probably have been added along the way but regrettably [the author](https://github.com/cpcallen) of those PRs neither thought to do so nor knew how to.

Reply:
OK, I was definitely under the impression that importing `blockly/blocks` got you the `Blockly.Blocks` library object but loaded with the library blocks into it already. If that's not the case. I don't think we currently have type definitions for anything exported under the `libraryBlocks` modules, so I don't know what to put here (as again it was outside the scope of this to add new definitions from scratch, I'm just trying to resolve the errors). I think we'd get actual type definitions when we move fully to TS.

So for now I can define `libraryBlocks` as ... something, which will allow people to use `Blockly.libraryBlocks` but won't provide deeper type checks? `Blockly.Blocks` will continue to work as that type is defined in core.